client/influxdb.go

```diff
@@ -624,6 +628,17 @@ func (c *Client) Addr() string {
 	return c.url.String()
 }
 
+// checkPointTypes ensures no unsupported types are submitted to influxdb, returning error if they are found.
+func checkPointTypes(p Point) error {
+	for _, v := range p.Fields {
+		switch v.(type) {
+		case uint64:
```

Comment:
Would it be better to explicitly catch what we do support (https://github.com/influxdb/influxdb/blob/master/models/points.go#L1310-L1353) and then default to something like 
```go
default:
    return fmt.Errorf("unsupported point type: %T", v)
```

That way it will catch all unsupported types, not just `uint64`.